docs/UserGuide.md

```diff
@@ -48,9 +48,9 @@ interactions recorded faster and simpler than traditional GUI applications.
 
    * **`list`** : Lists all companies.
 
-   * **`create`**`create 2 n/James p/97814456 e/test@test.com t/MainPoc t/owesMoney` : Creates a point-of-contact named `James` linked to the company at index 2.
+   * **`create`**`create 2 n/James p/97814456 e/test@test.com t/MainPoc t/owesMoney` : Creates a point-of-contact named `James` linked to the client at index 2.
 
-   * **`delete`**`delete 1` : Deletes the company at the first index together with all their contacts and transactions in the current list.
+   * **`delete`**`delete 1` : Deletes the client at the first index together with all their contacts and transactions in the current list.
 
    * **`find`**`MacDonalds` : Finds MacDonalds in the list of companies, and displays all its details (point-of-contact, transactions).
```

Comment:
Perhaps `companies` should be changed to clients?

Reply:
Okay sure thank you